backend/src/controllers/rides.ts

```diff
@@ -85,6 +85,18 @@ export const getRideById = async (req: CustomRequest, res: Response): Promise<vo
  * POST /rides
  * Create a new ride.
  */
+const fixTimeUpDayjs = () => {
```

Comment:
@aviv-taori No need to use dayjs here and for sure no need to add 3 hours as constant.